exec/java-exec/src/test/java/org/apache/drill/test/ClusterTest.java

```diff
@@ -125,4 +139,30 @@ public static void run(String query, Object... args) throws Exception {
   public QueryBuilder queryBuilder( ) {
     return client.queryBuilder();
   }
+
+  /**
+   * Generates a compressed version of the file for testing
+   * @param fileName Name of the input file
+   * @param codecName The desired CODEC to be used.
+   * @param outFileName Name of generated compressed file
+   * @throws IOException If function cannot generate file, throws IOException
+   */
+  public void generateCompressedFile(String fileName, String codecName, String outFileName) throws IOException {
```

Comment:
I like the idea of moving this. But, `ClusterTest` is perhaps the wrong place. Maybe create a utility class in the same package. (That way, it can also be use by `BaseTestQuery` and `OperatorFixture` based tests.)

Reply:
Fixed